docs/framework/configure-apps/file-schema/wcf/headers.md

```diff
@@ -14,8 +14,8 @@ An endpoint can be addressed by one or more SOAP headers in addition to its basi
   
 ```xml  
 <headers>
-  <region xmlns="Uri">"String"</region>
-  <member xmlns="Uri">"String"</member>
+  <Region xmlns="Uri">"String"</Region>
+  <Member xmlns="Uri">"String"</Member>
```

Comment:
Same fix here. Revert this change and instead remove the Child Elements table.

Reply:
Is it OK to remove the Child Elements section?